tests/test_client.py

```
@@ -344,7 +344,7 @@ async def test_json_decode_error(self) -> None:
 
         with (
             patch("unraid_mcp.core.client.get_http_client", return_value=mock_client),
-            pytest.raises(ToolError, match="Invalid JSON response"),
+            pytest.raises(ToolError, match="invalid response.*not valid JSON"),
```

Ruff (0.15.1)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)